app/views/home/index.html.erb

```diff
@@ -41,16 +41,15 @@
 		<!-- Right Announcements -->
 		<div class="flex -ml-16 relative z-20">
 			<div class="rounded-xl border-2 border-black overflow-hidden bg-white w-full flex flex-col">
-				<div class="bg-teal-500 text-white py-2 px-3 font-semibold text-sm tracking-wide border-b-2 border-black">Announcements</div>
-				<div class="p-4 text-xs text-gray-600 min-h-[150px] flex flex-col">
+				<div class="bg-teal-500 text-white py-1.5 px-3 font-semibold text-xs tracking-wide border-b-2 border-black">Announcements</div>
+				<div class="p-4 text-xs text-gray-600 max-h-[480px] overflow-y-auto">
```

Comment:
we have `overflow-y-auto` and `max-h-[480px]` here as an outer container, and we have again on the inside container. Is the outer container enough here?